### DISCUSSION TITLE
Можно ли реализовать поддержку взрослых СУБД(напр. PostgreSQL)?

### DISCUSSION THREAD

**Discussion by 00svd00** [petrovvlad/freeLib#98] [Ideas]
Собственно, вопрос в заголовке. Понятно что SQLite  в принципе закрывает потребности, но с полноценным PG можно будет натравить несколько инстансов программы на одну базу и сетевое хранилище. И обновлять централизованно

**Comment by petrovvlad** [comment]
Реализовать подключение к PostgreSQL несложно. Но если использовать одну БД для нескольких пользователей, их надо будет добавлять в БД, т.к там сохраняются метки (читаю, прочитано, и т.д.).

**Reply by 00svd00** [reply]
Благо, PG позволяет это делать легко и непринуждённо. И что самое приятное - эти метки уже не утеряются, т.к. живут на сервере

**Reply by petrovvlad** [reply]
Я имел ввиду, что метки должны быть персонифицированные. Т.е. в таблицы меток надо добавить поле "идентификатор пользователя". Как PG может облегчить это?

**Reply by 00svd00** [reply]
Пардон, не понял вопроса. Я пока не смотрел в sqlite базу как там что, но если метки - это отдельная таблица, тогда первый же костыль, который приходит в голову - для "клиенских" пользователей создаём отдельные схемы, в которых создаём вьюхи, смотрящие на главную таблицу с библиотекой. Ну а таблица с метками у каждого своя.
